doc/release/1.16.0-notes.rst

```diff
@@ -329,6 +329,10 @@ types. As of this release, this caveat is lifted - now:
   * Bitfields are no longer interpreted as sub-arrays
   * Pointers are no longer replaced with the type that they point to
 
+``matmul`` is now a ``ufunc``
+-----------------------------
+`numpy.matmul` is now a ufunc. It will use the same BLAS routines as
```

Comment:
Maybe add "which means that both the function and the ``__matmul__`` operator can now be overridden by ``__array_ufunc__``. Its implementation has also changed, ensuring it uses... , ensuring it its speed is similar for large matrices."